src/mam4xx/gas_chem.hpp

```diff
@@ -24,6 +24,46 @@ const Real rel_err = 1.0e-3;
 const Real high_rel_err = 1.0e-4;
 const int max_time_steps = 1000;
 
+KOKKOS_INLINE_FUNCTION
+void usrrxt(Real rxt[rxntot], // inout
+            const Real temp, const Real invariants[nfs], const Real mtot,
+            const int usr_HO2_HO2_ndx, const int usr_DMS_OH_ndx,
+            const int usr_SO2_OH_ndx, const int inv_h2o_ndx) {
+
+  /*-----------------------------------------------------------------
+   ... ho2 + ho2 --> h2o2
```

Comment:
There are lots of constants here, but at least it's easy to identify their purpose in a chemical kinetics calculation. Do you think we should mark up these constants, or leave them be?

Reply:
Yes, I believe we should mark them up. I added the "BAD CONSTANT" comment.